tests/foreman/ui_airgun/test_partitiontable.py

```diff
@@ -0,0 +1,211 @@
+
+
+def test_positive_create(session):
+    name = gen_string('alpha')
+    template = gen_string('alpha')
+    with session:
+        session.partitiontable.create(
+            {'name': name,
+             'default': True,
+             'template': template})
+        assert session.partitiontable.search(name)[0]['Name'] == name
+
+
+def test_positive_create_os_family(session):
+    name = gen_string('alpha')
+    template = gen_string('alpha')
+    with session:
+        session.partitiontable.create({
+            'name': name,
+            'snippet': False,
+            'os_family_selection': {'os_family': 'Debian'},
+            'template': template,
+        }, )
+        pt = session.partitiontable.read(name)
+        assert pt['os_family_selection']['os_family'] == 'Debian'
+        assert session.partitiontable.search(name)[0]['Name'] == name
+
+
+def test_positive_create_with_snippet(session):
+    name = gen_string('alpha')
+    template = gen_string('alpha')
+    with session:
+        session.partitiontable.create({
+            'name': name,
+            'snippet': True,
+            'template': template
+        }, )
+        assert session.partitiontable.search(name)[0]['Name'] == name
+
+
+def test_positive_create_with_audit_comment(session):
+    name = gen_string('alpha')
+    template = gen_string('alpha')
+    audit_comment = gen_string('alpha')
+    with session:
+        session.partitiontable.create({
+            'name': name,
+            'audit_comment': audit_comment,
+            'template': template
+        }, )
+        assert session.partitiontable.search(name)[0]['Name'] == name
+
+
+def test_positive_create_loc_org(session):
+    name = gen_string('alpha')
+    template = gen_string('alpha')
+    locations = entities.Location().create()
+    organizations = entities.Organization().create()
+    with session:
+        session.partitiontable.create({
+            'name': name,
+            'locations.assigned': [locations.name],
+            'organizations.assigned': [organizations.name],
+            'template': template
+        })
+        pt = session.partitiontable.read(name)
+        assert locations.name in pt['locations']['assigned']
+        assert organizations.name in pt['organizations']['assigned']
+
+
+def test_positive_delete_with_lock_and_unlock(session):
+    name = gen_string('alpha')
+    template = gen_string('alpha')
+    audit_comment = gen_string('alpha')
+    with session:
+        session.partitiontable.create({
+            'name': name,
+            'default': True,
+            'snippet': True,
+            'audit_comment': audit_comment,
+            'template': template
+        }, )
+        assert session.partitiontable.search(name)[0]['Name'] == name
+        session.partitiontable.lock(name)
+        try:
+            session.partitiontable.delete(name)
+        except ValueError:
+            assert session.partitiontable.search(name)[0]['Name'] == name
```

Comment:
Why try/except block? If you're trying to ensure deletion is not possible - you should use smth like this:
```python
from pytest import raises
# ...
with raises(ExceptionNameHere):
    session.partitiontable.delete(name)
```
where ExceptionNameHere - expected exception (not sure which one specifically, maybe NoSuchElementException or whatever is risen during deletion of locked ptable).